app/jobs/state_file/import_from_direct_file_job.rb

```diff
@@ -31,6 +31,7 @@ def perform(authorization_code:, intake:)
 
         intake.synchronize_df_dependents_to_database
         intake.synchronize_df_1099_rs_to_database
+        intake.synchronize_df_w2s_to_database
```

Comment:
This should also be called on `federal_info_form #save`, along with those other synchronize methods. Mike had pointed out that it's probably time to make a `synchronize_df_data_to_database` method that calls all of these since they are called in multiple places.